### PR TITLE
Fix external container-based tests JDK_VERSION setting

### PR DESCRIPTION
**Pull request by sophiaxu0424**
Related to issue: https://github.com/adoptium/aqa-tests/issues/5315

Following the steps to verify the mismatch and fix

### CODE REVIEW

**Comment by sophiaxu0424** [comment]
Grinder Tests: all passed for related testcases

**external/functional-test/example_functiona**l https://hyc-runtimes-jenkins.swg-devops.com/job/Grinder/42443  passed
**external/criu-functional** https://hyc-runtimes-jenkins.swg-devops.com/job/Grinder/42459/ passed
**external/criu-portable-checkpoint_test** https://hyc-runtimes-jenkins.swg-devops.com/job/Grinder/42461 passed
**external/criu-ubi-portable-checkpoint_test** https://hyc-runtimes-jenkins.swg-devops.com/job/Grinder/42462/ passed

**Comment by sophiaxu0424** [comment]
Hi @LongyuZhang @llxia would you please kind review, thanks!

File: external/dockerfile_functions.sh [hunk 1]
```diff
@@ -498,7 +498,7 @@ print_testInfo_env() {
     local test=$1
     local test_tag=$2
     local OS=$3
-    local version=$4
+    local version=$3
```
Comment (LongyuZhang) [inline external/dockerfile_functions.sh:501]: Both `OS` and `version` are assigned `$3`, we need to fix it here. Thanks.

Comment (sophiaxu0424) [reply external/dockerfile_functions.sh:501]: Yes, I do agree, that's what I suggested that we might need another issue for this OS_TAG parameter.

**Review by llxia — CHANGES_REQUESTED** [review]
As Longyu pointed out, it is wrong to assign `$3` to both OS and version. This is not the fix that we expected. We need to investigate why JDK version is set incorrectly.

**Comment by sophiaxu0424** [comment]
Passed Grinder test: https://hyc-runtimes-jenkins.swg-devops.com/job/Grinder/42544/

**Comment by smlambert** [comment]
@llxia - shall we close this draft?

**Comment by llxia** [comment]
close as obsolete.

**Pull request closed without merging**